cli/src/cast.rs

```diff
@@ -683,6 +690,56 @@ async fn main() -> eyre::Result<()> {
             let selector = HumanReadableParser::parse_function(&sig)?.short_signature();
             println!("0x{}", hex::encode(selector));
         }
+        Subcommands::Create2 { starts_with, ends_with, deployer, init_code } => {
```

Comment:
move to its own module and add an Args struct that bundles all fields here

Reply:
What do you mean by moving to its own module ?

Reply:
for example how the `find_block` command is organized https://github.com/foundry-rs/foundry/blob/master/cli/src/cmd/cast/find_block.rs

basically

make a `Create2Args struct` that bundles `{ starts_with, ends_with, deployer, init_code }`
then the `Subcommands::Create2` variant becomes `Create2(Create2Args)`

and the all of the cmd code can be moved to the `cli/src/cmd/cast/create2.rs` modules